elfi/methods/bo/utils.py

```diff
@@ -53,9 +55,93 @@ def sum_of_rbf_kernels(point, kern_centers, kern_ampl, kern_scale):
     return ret
 
 
-def stochastic_optimization(fun, bounds, maxiter=1000, polish=True):
+def stochastic_optimization(fun, bounds, maxiter=1000, polish=True, seed=0):
```

Comment:
Could we remove this? It is used only in `BayesianOptimization.extract_result` but that could be easily replaced with the minimize function or then just directly use this `differential_evolution` there